scipy/stats/morestats.py

```diff
@@ -1919,7 +1919,8 @@ def anderson_ksamp(samples, midrank=True):
     statistic : float
         Normalized k-sample Anderson-Darling test statistic.
     critical_values : array
-        The critical values for significance levels 25%, 10%, 5%, 2.5%, 1%.
+        The critical values for significance levels 25%, 10%, 5%, 2.5%, 1%,
+        0.5%, 0.1%.
```

Comment:
This looks good, but I think the documented floor of the `significance_level` mentioned below also needs to be updated now.

That is also mentioned in the linked issue. Once that is fixed, I think we can properly say that this "Fixes" the linked issue & have it automatically close once this is merged.

Reply:
Thanks for the catch!